docusaurus/docs/guides/kubernetes/workload-tunneling/index.md

```diff
@@ -38,34 +39,38 @@ In the chart above, there are several strategies and solutions for connecting a
 
 An OpenZiti Tunneler can be used to intercept pod egress. One important thing to know is that the OpenZiti Tunneler that is used in this way may also be used to "host" OpenZiti Services in order to provide ingress to cluster services.
 
-#### [Node Proxy Daemonset](./kubernetes-daemonset.md)
+<!-- #### [Node Proxy Daemonset](./kubernetes-daemonset.md)
 
-Deploying a daemonset of privileged `ziti-edge-tunnel run` pods on selected nodes is a simple way to enable OpenZiti services in a cluster. The daemonset pods intercept egress from pods and provide a DNS nameserver for CoreDNS. Like any other OpenZiti Tunneler, the OpenZiti Identity used by the daemonset may be configured to host OpenZiti Services, i.e. provide OpenZiti ingress to cluster services. For more information, see the [node proxy](./kubernetes-daemonset.md) page.
+Deploying a daemonset of privileged `ziti-edge-tunnel run` pods on selected nodes is a simple way to enable OpenZiti services in a cluster. The daemonset pods intercept egress from pods and provide a DNS nameserver for CoreDNS. Like any other OpenZiti Tunneler, the OpenZiti Identity used by the daemonset may be configured to host OpenZiti Services, i.e. provide OpenZiti ingress to cluster services. For more information, see the [node proxy](./kubernetes-daemonset.md) page. -->
 
-#### [Transparent Proxy Sidecar](./kubernetes-sidecar.md)
+#### [ziti-tunnel as Transparent Proxy Sidecar](./kubernetes-sidecar.md)
 
 You can deploy a transparent proxy sidecar to intercept pod egress. The sidecar provides a DNS nameserver that is used by the workload application to resolve OpenZiti Service addresses. The sidecar container runs `ziti tunnel tproxy` to create IPTables TPROXY rules in the pod. For more information, see the [transparent proxy sidecar](./kubernetes-sidecar.md) quickstart.
 
-#### [Loopback Proxy Sidecar](./kubernetes-sidecar.md)
+#### [ziti-tunnel as Loopback Proxy Sidecar](./kubernetes-sidecar.md)
 
 You can deploy a loopback proxy sidecar for pod egress to OpenZiti Services. The sidecar binds OpenZiti Services to a local port on the pod's loopback interface, e.g., 127.0.0.1:8443. The workload application must be configured to connect to the local port. This mode of operation has a few differences in comparison to the [transparent proxy sidecar](./kubernetes-sidecar.md) quickstart.
 
 1. The sidecar container runs `ziti tunnel proxy` instead of `ziti tunnel tproxy`.
 2. The sidecar container does not need the CAP_NET_ADMIN capability.
 3. The sidecar container does not need a `dnsPolicy` or explicit nameserver.
 
+#### [ziti-router as Cluster Service Proxy](/guides/kubernetes/hosting/kubernetes-router.mdx)
```

Comment:
i'd prefer removing `ziti-router` here too. not only is it not accurate, it's not adding anything to the reader's comprehension. If you feel strongly about it, at least change it to something like: "Router-based Cluster Service Proxy". I just don't think it adds much to put it into the title like this

Reply:
The name of the container image is `docker.io/openziti/ziti-router`, and the Helm chart is `openziti/ziti-router`. I meant to allude to those by a common name, but it might be better as you suggest to only refer to them generically like "Router-based" or "Ziti Router as ..."

Reply:
imo, it's generally better to be less specific rather than pedantic. it leads to fewer updates being needed and allows for subtle renames without having to rework stuff like this

Reply:
Alrighty. I'll change the headings to be more generic like "Router-based Proxy", "Tunnel-based Reverse Proxy", etc.